lib/tasks/sandbox_data.rake

```diff
@@ -1,83 +1,17 @@
 # frozen_string_literal: true
 
-namespace :lead_provider do
-  desc "create seed mentors for API testing"
-  task seed_mentors: :environment do
+require "tasks/valid_test_data_generator"
```

Comment:
I wonder if we should put it in `services` in some namespace. The fact that tasks are the only ones using it is true at the moment, but in principle I could see other things using it - say our seeds for review apps.

Reply:
Maybe later. I think that the TRN and URN generators could be used, for example, by factories.